openbas-api/src/main/java/io/openbas/utils/InjectUtils.java

```diff
@@ -97,4 +102,39 @@ public static boolean checkIfRowIsEmpty(Row row) {
     }
     return true;
   }
+
+  public static Inject duplicateInject(@NotNull Inject injectOrigin) {
+    ObjectMapper objectMapper = new ObjectMapper();
+    Inject duplicatedInject = new Inject();
+    duplicatedInject.setUser(injectOrigin.getUser());
+    duplicatedInject.setTitle(injectOrigin.getTitle());
+    duplicatedInject.setDescription(injectOrigin.getDescription());
+    try {
+      ObjectNode content =
+          objectMapper.readValue(
+              objectMapper.writeValueAsString(injectOrigin.getContent()), ObjectNode.class);
+      duplicatedInject.setContent(content);
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException(e);
+    }
+    duplicatedInject.setAllTeams(injectOrigin.isAllTeams());
+    duplicatedInject.setTeams(injectOrigin.getTeams().stream().toList());
+    duplicatedInject.setEnabled(injectOrigin.isEnabled());
+    duplicatedInject.setDependsDuration(injectOrigin.getDependsDuration());
+    if (injectOrigin.getDependsOn() != null) {
+      duplicatedInject.setDependsOn(injectOrigin.getDependsOn().stream().toList());
+    }
+    duplicatedInject.setCountry(injectOrigin.getCountry());
+    duplicatedInject.setCity(injectOrigin.getCity());
+    duplicatedInject.setInjectorContract(injectOrigin.getInjectorContract().orElse(null));
+    duplicatedInject.setAssetGroups(injectOrigin.getAssetGroups().stream().toList());
+    duplicatedInject.setAssets(injectOrigin.getAssets().stream().toList());
+    duplicatedInject.setCommunications(injectOrigin.getCommunications().stream().toList());
+    duplicatedInject.setPayloads(injectOrigin.getPayloads().stream().toList());
```

Comment:
can you remove that please?  because is not use anymore in my next PR i already remove it. You can check the table payloads_injects and see that is an empty table

Reply:
thanks for the heads up @MarineLeM ! I remove the line since it's dead code.